### PR TITLE
[GIE-IR] Fix the doc inconsistency issue of the legacy MaxGraph and the latest Gaia-IR. 

### PR DESCRIPTION
**Pull request by shirly121**
## What do these changes do?
as titled

## Related issue number

Fixes

### CODE REVIEW

File: docs/interactive_engine.rst [hunk 1]
```diff
@@ -249,121 +262,101 @@ Currently GIE supports the following Gremlin steps:
...
     g.V().values("name").groupCount()
-    
+
     //groupBy
     g.V().out().group()
     g.V().out().group().by("name")
     g.V().out().group().by().by("name")
-    
+
+    //groupBy multiple keys and set aliases
```
Comment (shirly121) [inline docs/interactive_engine.rst:328]: 这里加了group by multiple keys/values的一些用法

File: docs/interactive_engine.rst [hunk 2]
```diff
@@ -193,54 +194,66 @@ Currently GIE supports the following Gremlin steps:
...
+    //filter with a sub-query
     g.V().where(out().count().is(gt(4)))
-    
+
     //dedup
     g.V().out().dedup()
     g.V().out().dedup().by("name")
-    
-    //range
+    g.V().as("a").out().dedup("a").by("name")
```
Comment (shirly121) [inline docs/interactive_engine.rst:241]: 这里加上dedup('tag')

File: docs/interactive_engine.rst [hunk 3]
```diff
@@ -193,54 +194,67 @@ Currently GIE supports the following Gremlin steps:
...
     g.V().values("age").is(gt(70))
-    
-    //filter
-    g.V().filter(values("age").is(gt(20)))
-    
-    //where
+
+    //filter with expression (`expr()` syntactic sugar)
+    g.V().where(expr('@.age > 20')) //@.age refers to the "age" property of the head entry
+    g.V().as('a').out().as('b').where(expr('@a.age <= @b.age'))  //@a.age refers to the "age" property of the "a" entry
```
Comment (longbinlai) [inline docs/interactive_engine.rst:232]: expression这里增加几个bit操作的例子哈

**Comment by longbinlai** [comment]
In response, we add the doc for syntactic sugar of path expansion and expr. In addition, some new language feature such as text (string) compare (startswith, endswith) and  bitwise operations (bitand, bitor, bitxor, bit shift) are also supported.

**Review by sighingnow — COMMENTED** [review]
Thanks a lot for doing that!

We have a Chinese documentation as well, @shirly121 could you please sync the Chinese version with the English one?

File: docs/interactive_engine.rst (hunk 3 above)
Comment (shirly121) [reply docs/interactive_engine.rst:232]: added https://github.com/alibaba/GraphScope/pull/2074/commits/078ebf6f938487f4c68e5ac6df9b79ec3d313d7a

**Review by longbinlai — APPROVED** [review]

**Pull request merged**